Source/Core/DolphinQt2/GameList/GameListModel.cpp

```diff
@@ -138,6 +138,63 @@ int GameListModel::columnCount(const QModelIndex& parent) const
   return NUM_COLS;
 }
 
+bool GameListModel::ShouldDisplayGameListItem(const QModelIndex& index) {
+  QSharedPointer<GameFile> game = m_games[index.row()];
+
+
+  const bool show_platform = [&game] {
```

Comment:
Try capturing the platform ID, instead of passing the whole ``GameFile`` object into the lambda.